File: Frontend/IMMRequest/src/app/models/administrator.ts
export class Administrator {
    Id: string;
    Name: string;
    Email: string;
    Password: string;

    constructor(id: string, name: string, email: string, password: string) {
        this.Id = id;
        this.Name = name;
        this.Email = email;
        this.Password = password;
    }
}

File: Backend/IMMRequest/IMMRequest.Domain/Entities/Administrator.cs
using System;

namespace IMMRequest.Domain 
{
    public class Administrator
    {
        public Guid Id {get; set;}
        
        public string Name { get; set; }

        public string Email { get; set; }

        public string Password { get; set; }

        public Guid Token { get; set; }
        public Boolean IsDeleted { get; set; }

        public Administrator() {
            this.Id = Guid.NewGuid();
            this.Token = Guid.NewGuid();
            this.IsDeleted = false;
        }

        public Administrator(string Name, string Email, string Password) 
        {
            this.Id = Guid.NewGuid();
            this.Name = Name;
            this.Email = Email;
            this.Password = Password;
            this.Token = Guid.NewGuid();
            this.IsDeleted = false;
        }
    }
}
